app/initializers/ember-kiss-metrics.js

```diff
@@ -1,5 +1,10 @@
-export function initialize(container, application) {
-  container.options('kmq:main');
+export function initialize() {
+  let application = arguments[1] || arguments[0];
```

Comment:
sorry to be pedantic, but I believe a more accurate check is arity? `arguments.length === 1` vs `arguments.length === 2`